src/Data/Erased.agda

```diff
@@ -0,0 +1,43 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Wrapper for the erased modality
+-- This allows us to store erased proofs in a record an use projections
```

Comment:
Is the `an` a typo? Also as mentioned in #994 the second paragraph should probably go just below the header? (added this to the list of things to add to the style guide!)